libs/SalesforceSDK/res/values/sf__strings.xml

```diff
@@ -24,7 +24,7 @@
 	<string name="sf__pick_server">Change Server</string>
 	<string name="sf__clear_cookies">Clear Cookies</string>
 	<string name="sf__reload">Reload</string>
-	<string name="sf__launch_idp">Launch IDP App</string>
+	<string name="sf__launch_idp">Login with IDP App</string>
```

Comment:
Changed the label on the button since there is no swizzling to the IDP app anymore - we use the current user in the IDP app if any or return an error if the IDP app is not logged in.